codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/protocols/ServerHttpProtocolGenerator.kt

```diff
@@ -967,6 +967,7 @@ private class ServerHttpProtocolImplGenerator(
     private fun generateParsePercentEncodedStrAsStringFn(binding: HttpBindingDescriptor): RuntimeType {
         val output = symbolProvider.toSymbol(binding.member)
         val fnName = generateParseStrFnName(binding)
+        val returnVal = if (symbolProvider.toSymbol(binding.member).isOptional()) { "Some(value)" } else { "value" }
```

Comment:
this can be `wrapOptional` now I think

Reply:
Yes! Old habits, I need to get used to the two new methods :(